2021-05-19 18:39:10,784:DEBUG:certbot._internal.main:certbot version: 1.15.0
2021-05-19 18:39:10,785:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2021-05-19 18:39:10,785:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/usr/share/nginx/html/letsencrypt', '--email', 'robotixclub@nitrr.ac.in', '--agree-tos', '--no-eff-email', '-d', 'roborix.nitrr.ac.in']
2021-05-19 18:39:10,785:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2021-05-19 18:39:10,884:DEBUG:certbot._internal.log:Root logging level set at 20
2021-05-19 18:39:10,884:INFO:certbot._internal.log:Saving debug log to /var/log/letsencrypt/letsencrypt.log
2021-05-19 18:39:10,894:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2021-05-19 18:39:10,910:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Place files in webroot directory
Interfaces: IAuthenticator, IPlugin
Entry point: webroot = certbot._internal.plugins.webroot:Authenticator
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f4068ec61c0>
Prep: True
2021-05-19 18:39:10,910:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f4068ec61c0> and installer None
2021-05-19 18:39:10,910:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2021-05-19 18:39:11,361:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2021-05-19 18:39:11,368:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2021-05-19 18:39:12,357:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 658
2021-05-19 18:39:12,358:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 19 May 2021 18:39:12 GMT
Content-Type: application/json
Content-Length: 658
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "lUGJOYFTQWs": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2021-05-19 18:39:12,358:DEBUG:acme.client:Requesting fresh nonce
2021-05-19 18:39:12,358:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2021-05-19 18:39:12,605:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2021-05-19 18:39:12,606:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 19 May 2021 18:39:12 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0003KRQ3HA-bvZ5Sy_XCnDi2wQkj7UNK1vWK22umH99npbc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2021-05-19 18:39:12,606:DEBUG:acme.client:Storing nonce: 0003KRQ3HA-bvZ5Sy_XCnDi2wQkj7UNK1vWK22umH99npbc
2021-05-19 18:39:12,606:DEBUG:acme.client:JWS payload:
b'{\n  "contact": [\n    "mailto:robotixclub@nitrr.ac.in"\n  ],\n  "termsOfServiceAgreed": true\n}'
2021-05-19 18:39:12,609:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-acct:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAiandrIjogeyJuIjogInlFZ242X0MydzlBUVVtMDBOcUozeGozN3UzWW9KT24wZ1hpYnVuVk40bkhHbGpXR0phSy1NYjNwN1BWTHJ0anE4RE54NnN6aUtoLUhMUm54VGRUZk1zQUFyTGthQzZ4TVdZLWhNV3B6ZUR1dVIzRTRSaVFJVW9VdXR1TGg0WHY4M0o3M1o4eHpQVnk3VG5iQ1pVM0t0ZUZMU0Zka0Y5TWRLbmphYi0wLTNSR3h4a0lxbWZETkFRWWgxT2FmV3pQZjk3R2wxUFN4SFJNeFNWVFN2LXc1cUJZTkYtV0l6bGtLa25WbWlJc2xlYVFLZWh5WjBRWmVVcHJMTUFwSkctVnp3b2FfODVsWlJQbUcyMjl0SFNUaXh0Mi0zTlVWN2VxTFFnenJRQmxUdEZWalkyMm55dFNpQk5tcEJNNkREZGhqbDF4T3N5MldfNmRUNFp2aEJyRDVHUSIsICJlIjogIkFRQUIiLCAia3R5IjogIlJTQSJ9LCAibm9uY2UiOiAiMDAwM0tSUTNIQS1idlo1U3lfWENuRGkyd1FrajdVTksxdldLMjJ1bUg5OW5wYmMiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1hY2N0In0",
  "signature": "eZUjl2e_0wGONb7EbVepmN52f2mnTInEnoV16yEFi5mS7nhyt1V4thQOtlWPQzysEXiZYxkfUNewlymgBLR_Wilg27FG-qny3lhaA7hnruS0SZKc-ToAjM_EYI_XDBQ9nUwCXtiwB9LBdRfM31djEtQe-Puzmg_ApiASURUtYf_V1vCe8YhP248bwGagpbzDN0Dvn49QcRzD9zUVMeSm0I9vplPhYEzzk1hteitPKMSYVc4f8OLl-xmVX_sbXtUCTRIyokQQ4250jltMaS1m9fkB--xfhmq9HEgY5RzJwLdLp5bnOon1oU6TF0XSQR0ispyeMwA0z5fXcjvFuTIv_w",
  "payload": "ewogICJjb250YWN0IjogWwogICAgIm1haWx0bzpyb2JvdGl4Y2x1YkBuaXRyci5hYy5pbiIKICBdLAogICJ0ZXJtc09mU2VydmljZUFncmVlZCI6IHRydWUKfQ"
}
2021-05-19 18:39:12,930:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-acct HTTP/1.1" 201 564
2021-05-19 18:39:12,931:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Wed, 19 May 2021 18:39:12 GMT
Content-Type: application/json
Content-Length: 564
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf>;rel="terms-of-service"
Location: https://acme-v02.api.letsencrypt.org/acme/acct/124178225
Replay-Nonce: 0004IcqfE7JPOlqgXbpPaP1qD-oco1m4C_-ws6v-_zm1AD8
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "key": {
    "kty": "RSA",
    "n": "yEgn6_C2w9AQUm00NqJ3xj37u3YoJOn0gXibunVN4nHGljWGJaK-Mb3p7PVLrtjq8DNx6sziKh-HLRnxTdTfMsAArLkaC6xMWY-hMWpzeDuuR3E4RiQIUoUutuLh4Xv83J73Z8xzPVy7TnbCZU3KteFLSFdkF9MdKnjab-0-3RGxxkIqmfDNAQYh1OafWzPf97Gl1PSxHRMxSVTSv-w5qBYNF-WIzlkKknVmiIsleaQKehyZ0QZeUprLMApJG-Vzwoa_85lZRPmG229tHSTixt2-3NUV7eqLQgzrQBlTtFVjY22nytSiBNmpBM6DDdhjl1xOsy2W_6dT4ZvhBrD5GQ",
    "e": "AQAB"
  },
  "contact": [
    "mailto:robotixclub@nitrr.ac.in"
  ],
  "initialIp": "3.108.0.154",
  "createdAt": "2021-05-19T18:39:12.800618524Z",
  "status": "valid"
}
2021-05-19 18:39:12,931:DEBUG:acme.client:Storing nonce: 0004IcqfE7JPOlqgXbpPaP1qD-oco1m4C_-ws6v-_zm1AD8
2021-05-19 18:39:12,934:DEBUG:certbot.display.util:Notifying user: Account registered.
2021-05-19 18:39:12,934:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=JWKRSA(key=<ComparableRSAKey(<cryptography.hazmat.backends.openssl.rsa._RSAPublicKey object at 0x7f4068ebe400>)>), contact=('mailto:robotixclub@nitrr.ac.in',), agreement=None, status='valid', terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/124178225', new_authzr_uri=None, terms_of_service='https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf'), 4ccf1848dab134df4a9a7b01a72cbb83, Meta(creation_dt=datetime.datetime(2021, 5, 19, 18, 39, 12, tzinfo=<UTC>), creation_host='30d663a1f659', register_to_eff=None))>
2021-05-19 18:39:12,935:DEBUG:certbot.display.util:Notifying user: Requesting a certificate for roborix.nitrr.ac.in
2021-05-19 18:39:13,009:DEBUG:certbot.crypto_util:Generating RSA key (2048 bits): /etc/letsencrypt/keys/0000_key-certbot.pem
2021-05-19 18:39:13,014:DEBUG:certbot.crypto_util:Creating CSR: /etc/letsencrypt/csr/0000_csr-certbot.pem
2021-05-19 18:39:13,015:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "roborix.nitrr.ac.in"\n    }\n  ]\n}'
2021-05-19 18:39:13,017:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDRJY3FmRTdKUE9scWdYYnBQYVAxcUQtb2NvMW00Q18td3M2di1fem0xQUQ4IiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9uZXctb3JkZXIifQ",
  "signature": "epfYpStDvDgk--TrUn4EOhnyXoIyO-rDjrkedGfi7QttFZwaMecSfqGhsBsk0HT5lNkrkN_xBZ_qcEh8A3Ni6fo85jhCLSlcetLl_UwmdolRBZdA9dewaQCsCRaq6VmGiR1ZN-EbFPUIME0oiU_7UgXq9v13wMtYnWji_TSzvrWWfVJddt_jTiUkQbnN3QMnS4pN4NHnkJsHBxIFpbhZ5XOuNp80LEGUVaRjVp1vbq4qacj2z9XTNfHdzHyylKL14NoFQCodXXu2IsSv7AoFaH15hrQ_hiX1x29ymiak2SH_lvqz3J2FLX8TZ4QEayz_ninaee4BxNVRzRVWHN_qkg",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInJvYm9yaXgubml0cnIuYWMuaW4iCiAgICB9CiAgXQp9"
}
2021-05-19 18:39:13,422:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 341
2021-05-19 18:39:13,423:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Wed, 19 May 2021 18:39:13 GMT
Content-Type: application/json
Content-Length: 341
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/124178225/9812493943
Replay-Nonce: 0003Ojt9RSstoLSN5j8jR3zL0Bi__Fi_Z_6iYxAi2VRIqJ8
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2021-05-26T18:39:13Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "roborix.nitrr.ac.in"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/13264622912"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/124178225/9812493943"
}
2021-05-19 18:39:13,423:DEBUG:acme.client:Storing nonce: 0003Ojt9RSstoLSN5j8jR3zL0Bi__Fi_Z_6iYxAi2VRIqJ8
2021-05-19 18:39:13,424:DEBUG:acme.client:JWS payload:
b''
2021-05-19 18:39:13,429:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/13264622912:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDNPanQ5UlNzdG9MU041ajhqUjN6TDBCaV9fRmlfWl82aVl4QWkyVlJJcUo4IiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My8xMzI2NDYyMjkxMiJ9",
  "signature": "UVyr9n_-5KHEHH1lrKqplX1AcEMm1gp6_wTonroQ-9Ef1tcf59CIe-qWF3H-zlhj4dODcB9VoxZ2_wqdcF3--p0MwTEHR5dsq85u16xs1N4_9Bst5URFYJbdZmpCgB7Z9hOYqaYyIEWiu2cJ_pOxYbMOyF_j3YB9aVcmTwcmTAQ8JCrUkZ_QjnseIH_UrDDjE5_W6iBGqLQLua_dez1Sj8qKkctuw1XvO_-C7US8prM1tvznVN6WYCeLA6vC-ZZnGxcN0OHfa5HJBa1YaUSUe0T6mlrNTzryCVCJ3Dku8Jguovv2K3LzvMRT3oO6dPv-7lD7XU9X6l8q7lva5L_I1Q",
  "payload": ""
}
2021-05-19 18:39:13,714:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13264622912 HTTP/1.1" 200 800
2021-05-19 18:39:13,715:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 19 May 2021 18:39:13 GMT
Content-Type: application/json
Content-Length: 800
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 00032sOI28w5zwgLv1Jed3nMFTQoJucN7af_MQjpRvgCRVk
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "roborix.nitrr.ac.in"
  },
  "status": "pending",
  "expires": "2021-05-26T18:39:13Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13264622912/vXERyA",
      "token": "WoACFL8IxO9ZEsN1r6tfHvYrf0ryVDMouw9_y59zIA8"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13264622912/zKKmsQ",
      "token": "WoACFL8IxO9ZEsN1r6tfHvYrf0ryVDMouw9_y59zIA8"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13264622912/UKKVKA",
      "token": "WoACFL8IxO9ZEsN1r6tfHvYrf0ryVDMouw9_y59zIA8"
    }
  ]
}
2021-05-19 18:39:13,715:DEBUG:acme.client:Storing nonce: 00032sOI28w5zwgLv1Jed3nMFTQoJucN7af_MQjpRvgCRVk
2021-05-19 18:39:13,715:INFO:certbot._internal.auth_handler:Performing the following challenges:
2021-05-19 18:39:13,721:INFO:certbot._internal.auth_handler:http-01 challenge for roborix.nitrr.ac.in
2021-05-19 18:39:13,721:INFO:certbot._internal.plugins.webroot:Using the webroot path /usr/share/nginx/html/letsencrypt for all unmatched domains.
2021-05-19 18:39:13,721:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge
2021-05-19 18:39:13,723:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/WoACFL8IxO9ZEsN1r6tfHvYrf0ryVDMouw9_y59zIA8
2021-05-19 18:39:13,723:INFO:certbot._internal.auth_handler:Waiting for verification...
2021-05-19 18:39:13,724:DEBUG:acme.client:JWS payload:
b'{}'
2021-05-19 18:39:13,732:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/13264622912/vXERyA:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDMyc09JMjh3NXp3Z0x2MUplZDNuTUZUUW9KdWNON2FmX01RanBSdmdDUlZrIiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9jaGFsbC12My8xMzI2NDYyMjkxMi92WEVSeUEifQ",
  "signature": "XTZErDvmASCPcoVVbSBIXsia3Nns1GFrrCmjUSZLgUGv0tBE1IDTgE-CbZJmxP3WJdArhZ-IdWj4gGqM2k5B0g4wE-Cf1F6fjWWiJzJ0n2yb26MwcnuhqtJCBMFFrpr5EL0msL-tUyCO-OYU0BlY-McsJtgiQKaJkB7STGhl0KYoDWZHMIY56Z5Ivcry-eDh3dyURpvnLN9v2g7qnnbkbC97AFvwbFW4Zqt_ekOqygAu28QXbp3WLhzDUdowQGRZeQd4YL1y8jquqNZV6EtDwVYgtqFC64DfA7bqskBkZHV2DTN4YcS-ABglvmrz2ZokqXhr_UkGmQzi9MGuOQ5AHg",
  "payload": "e30"
}
2021-05-19 18:39:14,030:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/13264622912/vXERyA HTTP/1.1" 200 186
2021-05-19 18:39:14,030:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 19 May 2021 18:39:13 GMT
Content-Type: application/json
Content-Length: 186
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/13264622912>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/13264622912/vXERyA
Replay-Nonce: 0003ax0jIS2ZCMCMm51iDf_9wRVT6czQ8ACK3uJE_3VerjM
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13264622912/vXERyA",
  "token": "WoACFL8IxO9ZEsN1r6tfHvYrf0ryVDMouw9_y59zIA8"
}
2021-05-19 18:39:14,031:DEBUG:acme.client:Storing nonce: 0003ax0jIS2ZCMCMm51iDf_9wRVT6czQ8ACK3uJE_3VerjM
2021-05-19 18:39:15,034:DEBUG:acme.client:JWS payload:
b''
2021-05-19 18:39:15,036:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/13264622912:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDNheDBqSVMyWkNNQ01tNTFpRGZfOXdSVlQ2Y3pROEFDSzN1SkVfM1ZlcmpNIiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My8xMzI2NDYyMjkxMiJ9",
  "signature": "WCV3ppX89IYI08FrLGZX43o952qXsUdYlTqLjsSTNIbX1fndC0tMvCNwIleEYUT2X7cvMUzX9X_qv3TnTXjkgdaAEWr0sJwVvL9kTahyML53G4zfyaUmOLUIMAnJN2eNcRfO9W6qMBSOvGv9agzdnwU-CNvsELQ2qTGbTj67jArjhnd-vHC9IrS055Y6HNxwgOvvxrucLJ0JJZYLfiVTT9IEURJtjAutTLc4Vkh_CcarulP2kXhpxnY6c-aHkcf-ofvplj3ZqZhev6UCMn9fGyUH7QP4Ouf2JGMtB0PHOxncFkJFAQXT6AmHNcD5X-O7WotpandQiYxrIEmgUdMy8w",
  "payload": ""
}
2021-05-19 18:39:15,308:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13264622912 HTTP/1.1" 200 643
2021-05-19 18:39:15,309:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 19 May 2021 18:39:15 GMT
Content-Type: application/json
Content-Length: 643
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 000364ylSx4xaDFfEkRZ8fPjpk7ANN3Djp0pwQbkw1xoy94
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "roborix.nitrr.ac.in"
  },
  "status": "invalid",
  "expires": "2021-05-26T18:39:13Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:dns",
        "detail": "DNS problem: NXDOMAIN looking up A for roborix.nitrr.ac.in - check that a DNS record exists for this domain",
        "status": 400
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13264622912/vXERyA",
      "token": "WoACFL8IxO9ZEsN1r6tfHvYrf0ryVDMouw9_y59zIA8",
      "validated": "2021-05-19T18:39:13Z"
    }
  ]
}
2021-05-19 18:39:15,310:DEBUG:acme.client:Storing nonce: 000364ylSx4xaDFfEkRZ8fPjpk7ANN3Djp0pwQbkw1xoy94
2021-05-19 18:39:15,310:WARNING:certbot._internal.auth_handler:Challenge failed for domain roborix.nitrr.ac.in
2021-05-19 18:39:15,311:INFO:certbot._internal.auth_handler:http-01 challenge for roborix.nitrr.ac.in
2021-05-19 18:39:15,312:DEBUG:certbot._internal.reporter:Reporting to user: The following errors were reported by the server:

Domain: roborix.nitrr.ac.in
Type:   dns
Detail: DNS problem: NXDOMAIN looking up A for roborix.nitrr.ac.in - check that a DNS record exists for this domain
2021-05-19 18:39:15,312:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 179, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2021-05-19 18:39:15,312:DEBUG:certbot._internal.error_handler:Calling registered functions
2021-05-19 18:39:15,312:INFO:certbot._internal.auth_handler:Cleaning up challenges
2021-05-19 18:39:15,314:DEBUG:certbot._internal.plugins.webroot:Removing /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/WoACFL8IxO9ZEsN1r6tfHvYrf0ryVDMouw9_y59zIA8
2021-05-19 18:39:15,314:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2021-05-19 18:39:15,314:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 15, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1435, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1304, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 140, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 444, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 374, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 424, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 179, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2021-05-19 18:39:15,315:ERROR:certbot._internal.log:Some challenges have failed.
